src/cli/parser.rs

```diff
@@ -69,6 +69,46 @@ lazy_static! {
     )
     .unwrap();
 
+    // rgb(255 0 119)
+    pub static ref RE_RGB_SPACE: Regex = Regex::new(
+        r"(?x)
+            ^
+            rgb\(
+                \s*
+                (\d{1,3})
+                \s*
```

Comment:
Shouldn't we have `\s+` as a separator between the numbers? Could you please add a test that `rgb(2550119)` is *not* parsed as a color?

Reply:
Yes you're right!